packages/components/psammead-navigation/src/index.jsx

```diff
@@ -0,0 +1,214 @@
+
+    @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+      top: -3rem;
+      left: 0.5rem;
+    }
+  }
+`;
+
+const StyledUnorderedList = styled.ul`
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+
+  /* Display a transparent border line > 600px */
+  &::after {
+    content: ' ';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    border-bottom: 0.0625rem solid transparent;
+    z-index: 1;
+  }
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    &::after {
+      content: none;
+    }
+  }
+`;
+
+const StyledListItem = styled.li`
+  display: inline-block;
+  position: relative;
+  z-index: 2;
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    /* Trick to display a border between the list items, which takes the full width */
+    &::after {
+      content: ' ';
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN};
+      border-bottom: 1px solid ${BORDER_COLOR};
+      z-index: -1;
+
+      ${({ dir }) =>
+        dir === 'ltr'
+          ? css`
+              left: 0;
+            `
+          : css`
+              right: 0;
+            `};
+    }
+  }
+`;
+
+const ListItemBorder = css`
+  content: '';
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-bottom: 0.25rem solid ${C_WHITE};
+`;
+
+const StyledLink = styled.a`
+  ${props => (props.script ? getPica(props.script) : '')};
+  font-family: ${GEL_FF_REITH_SANS};
+  color: ${C_GHOST};
+  cursor: pointer;
+  text-decoration: none;
+  display: inline-block;
+  padding: 0.75rem ${GEL_SPACING};
+
+  &:hover::after {
+    ${ListItemBorder}
+  }
+
+  &:focus::after {
+    ${ListItemBorder}
+    top: 0;
+    border: 0.25rem solid ${C_WHITE};
+  }
+`;
+
+const StyledSpan = styled.span`
+  ${props => (props.script ? getPica(props.script) : '')};
+  font-family: ${GEL_FF_REITH_SANS};
+  color: ${C_GHOST};
+  display: inline-block;
+  padding: 0.75rem ${GEL_SPACING};
+
+  &::before {
+    ${ListItemBorder}
+  }
+`;
+
+const CurrentItem = ({ children: link, script }) => (
+  <Fragment>
+    <StyledSpan
+      // eslint-disable-next-line jsx-a11y/aria-role
+      role="text"
+      script={script}
+    >
+      <VisuallyHiddenText>Current page,</VisuallyHiddenText>
+      {link}
+    </StyledSpan>
+  </Fragment>
+);
+
+export const NavigationUl = ({ children, ...props }) => (
+  <StyledUnorderedList role="list" {...props}>
+    {children}
+  </StyledUnorderedList>
+);
+
+export const NavigationLi = ({ children: link, url, script, dir, active }) => (
+  <StyledListItem role="listitem" dir={dir}>
+    {active ? (
+      <CurrentItem script={script}>{link}</CurrentItem>
+    ) : (
+      <StyledLink href={url} script={script}>
+        {link}
+      </StyledLink>
+    )}
+  </StyledListItem>
+);
+
+const Navigation = ({ children, skipLinkText }) => (
+  <StyledNav role="navigation">
+    <SkipLink href="#content">{skipLinkText}</SkipLink>
```

Comment:
Do we have an issue to add `#content` to simorghs `main` tag?

Reply:
Yes, we have one [issue](https://github.com/bbc/simorgh/issues/1968) for this and it will link to the `h1`, not to the main tag.